### ISSUE TITLE
Cannot find module '../../config' - Only happens on the "production" deployment

### ISSUE THREAD

**Issue by dreaddymck** [helmetjs/helmet#178]
This is an odd one.

Happens on
 
` const helmet 		= require('helmet');`

Only happens on the production deployment. 
The production environment is a direct copy of development which includes /node_modules folder.
There is no npm install on production.

Still debugging. 

Leaving the output here just in case someone else is experiencing this.

```
Debugger listening on ws://0.0.0.0:9229/0e8316b7-bcfb-4ea7-99cb-7484b0738515
For help, see: https://nodejs.org/en/docs/inspector
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module '../../config'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/dreaddy/Production/socache-server/node_modules/helmet-csp/lib/check-options/check-directive/index.js:1:76)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

**Comment by EvanHahn** [comment]
Strange.

What version of Helmet or `helmet-csp` are you using?

Is there a `config.js` in `/home/dreaddy/Production/socache-server/node_modules/helmet-csp/lib/`?

**Comment by dreaddymck** [comment]
> What version of Helmet or helmet-csp are you using?

"helmet": "^3.13.0",

> Is there a config.js in /home/dreaddy/Production/socache-server/node_modules/helmet-csp/lib/?

I did not see one. 
I recently notice similar errors pop up in the other projects while developing. They've all been in some way associated with changes to the modules.

I ran into a out of sync module on that production server. Fixed it, but have yet to re-apply helmet to see if the problem was resolved.

Will keep you posted.
